lib/frame.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+class Frame
+  TEN_PINS = 10
+  FIRST_ROLL = 0
+  SECOND_ROLL = 1
```

Comment:
Nice use of constants to keep the code clear and obvious as to what is being referenced